src/module/documents/actor.mjs

```diff
@@ -2,6 +2,14 @@
  * A document subclass adding system-specific behavior and registered in CONFIG.Actor.documentClass
  */
 export default class DrawSteelActor extends foundry.documents.Actor {
+  /**
+   * Is this actor a minion?
+   * @returns {boolean}
+   */
+  get isMinion() {
+    return !!this.system.isMinion;
```

Comment:
I know this is a dnd5e'ism, but I have a preference for nullish operators here:
```suggestion
    return this.system.isMinion ?? false;
```
I feel like it also cleanly displays the default value (`false` in this case).